examples/jsm/renderers/nodes/core/InputNode.js

```diff
@@ -0,0 +1,38 @@
+import Node from './Node.js';
+
+class InputNode extends Node {
+
+	constructor( type ) {
+		
+		super( type );
+		
+		this.isInputNode = true;
```

Comment:
During the class migration, it was decided to write the `is*` properties like so:
```js
Object.defineProperty( this, 'isInputNode', { value: true } );
```